client/settings/payment-method-settings/index.js

```diff
@@ -13,7 +13,13 @@ import GiropaySettings from './giropay-settings';
 const methods = {
 	woocommerce_payments_giropay: {
 		title: 'giropay',
```

Comment:
Since the title is now in `description`, this should probably go. :)

Reply:
Sorry, can you clarify? It looks like the `title` attribute is still used to render the breadcrumbs on L48

Reply:
Yep, my mistake! Good catch!